src/diagnostics.rs

```diff
@@ -116,7 +117,17 @@ pub fn report_error<'tcx, 'mir>(
 
     e.print_backtrace();
     let msg = e.to_string();
-    report_msg(ecx, &format!("{}: {}", title, msg), msg, helps, true);
+    report_msg(*ecx.tcx, /*error*/true, &format!("{}: {}", title, msg), msg, helps, &ecx.generate_stacktrace());
```

Comment:
Now I gotta teach vscode not to render this as `/* error */ error: true` :laughing: